pages/index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Michelle's Portfolio Website</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
+
 <body>
-  
+<content id="main">
+<header>
+  <iframe width="0" height="0" src="https://www.youtube.com/embed/Atp8GPADPDM?controls=0?rel=0&autoplay=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
It's a little rude to autoplay audio on a web page, and not provide any UI for turning it off.